docs/sql/grammar.md

```diff
@@ -189,6 +191,27 @@ to `GROUP BY`.
 
 `MINUS` is equivalent to `EXCEPT`.
 
+### Grouping functions
+
+<table>
+  <tr>
+    <th>Function</th>
+    <th>Description</th>
+  </tr>
+  <tr>
+     <td><code>GROUPING(</code>expression, [ expression ]<code>)</code></td>
+     <td>Returns a bit vector of the grouping expressions</td>
```

Comment:
I was not able to follow this section. Maybe an example would help. For example what is a 

> bit vector of the grouping expressions

Reply:
these are not fully supported, I will improve the documentation when I finalize them